spec/feature/post_index_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Render post index page', type: :feature do
+  before :each do
+    @user = User.create(Name: 'Adhola', Photo: 'https://www.linkedtour.com/images/lkavatar.png',
+                        Bio: 'I am Micronaut, studying full stack development', PostsCounter: 0)
+    @first_post = Post.create(Title: 'Microverse',
+                              Text: 'Microverse online teaching school for those who want to be developers',
+                              CommentsCounter: 0, LikesCounter: 10, author: @user)
+    @second_post = Post.create(Title: 'Microverse', Text: 'The same post like the first one', CommentsCounter: 0,
+                               LikesCounter: 5,
+                               author: @user)
+    5.times do |_i|
+      Comment.create(Text: 'Nice post!!', author_id: @user.id, post_id: @first_post.id)
+    end
+    visit "/users/#{@user.id}/posts"
+  end
+
+  scenario 'displays post text' do
+    expect(page).to have_content('Microverse online teaching school for those who want to be developers')
+  end
+
+  scenario 'display the post title' do
+    expect(page).to have_content(@first_post.Title)
+  end
+
+  scenario 'display the post body' do
+    expect(page).to have_content(@first_post.Text)
+  end
+
+  scenario 'display the first comment on a post' do
+    expect(page).to have_content('Microverse')
+  end
+
```

Comment:
- [ ] Please make sure to write the below tests for the post index page as requested in the project requirements.
  - `Tests that asserts you can see the user's profile picture.`
  - `Tests that asserts you can see the user's username.`
  - `Tests that asserts you can see the number of posts the user has written.`